src/seamless-immutable.js

```diff
@@ -5,11 +5,36 @@
   var REACT_ELEMENT_TYPE = typeof Symbol === 'function' && Symbol.for && Symbol.for('react.element');
   var REACT_ELEMENT_TYPE_FALLBACK = 0xeac7;
 
+  function instantiateEmptyObject(obj) {
+      var prototype = Object.getPrototypeOf(obj);
+      if (!prototype) {
+          return {};
+      } else {
+          return Object.create(prototype);
+      }
+  }
+
+  function staticWarning(fn, name) {
+      function staticWrapper() {
+          console.log('DEPRECATED: Using x.' + name + '(...) will be deprecated. Please use Immutable.' + name + '(x, ...) instead.');
```

Comment:
It would be nice to use console.warn instead.

Also, instead of "will be deprecated", maybe it would be better to say "has been deprecated". Saying "DEPRECATED: ...will be deprecated" is a bit confusing.